compiler-rt/lib/sanitizer_common/sanitizer_linux_libcdep.cpp

```diff
@@ -1071,6 +1071,16 @@ void InitializePlatformCommonFlags(CommonFlags *cf) {
 #  endif
 }
 
+#  if SANITIZER_LINUX
+bool ShouldTreatRuntimeSecurely() {
+  // So we can use the weak definition from sanitizer_getauxval.h
+  if (&getauxval)
+    return getauxval(/* AT_SECURE */ 23) != 0;
+  else
```

Comment:
nit: get rid of the else (https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return)